app/models/comment.rb

```diff
@@ -0,0 +1,17 @@
+class Comment < ActiveRecord::Base
+  belongs_to :post, foreign_key: 'post_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
```

Comment:
- [ ] It would be nice if you update your associtions here as well since you have to update the associations in the user model 👍 You can make the assocations here as following ⬇️

```ruby
  belongs_to :post, foreign_key: 'post_id'
  belongs_to :user
```